src/components/Post/GovernanceSideBar/index.tsx

```diff
@@ -75,8 +75,6 @@ import executeTx from '~src/util/executeTx';
 import getAccountsFromWallet from '~src/util/getAccountsFromWallet';
 import Web3 from 'web3';
 import { useTheme } from 'next-themes';
-import PredictionCard from '~src/ui-components/PredictionCard';
```

Comment:
is this deletion needed?

Reply:
was getting eslint errors 'PredictionCard' is defined but never used.